cpp/src/aztec3/circuits/rollup/base/c_bind.cpp

```diff
@@ -66,7 +69,7 @@ WASM_EXPORT size_t base_rollup__sim(uint8_t const* base_rollup_inputs_buf,
     BaseRollupInputs<NT> base_rollup_inputs;
     read(base_rollup_inputs_buf, base_rollup_inputs);
 
-    BaseOrMergeRollupPublicInputs<NT> public_inputs = base_rollup_circuit(base_rollup_inputs);
+    BaseOrMergeRollupPublicInputs<NT> public_inputs = base_rollup_circuit(composer, base_rollup_inputs);
```

Comment:
Note to self: look at how Aztec Connect informed the Typescript that a circuit failed. I imagine we might want to return some more info (at least a success/fail bool) from each of these c_binds. I'll open an issue for this.

Reply:
Issue opened :)